Loki cloud integration instructions (and necessary mixin changes) (#8492)
grafana/loki: docs/sources/installation/helm/monitor-and-alert/with-local-monitoring.md

# Configure monitoring and alerting

> By default this Helm Chart configures meta-monitoring of metrics (service monitoring) and logs (self monitoring). This guide will walk you through configuring monitoring using a monitoring solution local to the same cluster Loki is installed in.

Reviewer: ```suggestion
By default this Helm Chart configures meta-monitoring of metrics (service monitoring) and logs (self monitoring). This topic will walk you through configuring monitoring using a monitoring solution local to the same cluster where Loki is installed.
```
